internal/ingress/controller/controller_test.go

```diff
@@ -253,7 +253,7 @@ func TestCheckIngress(t *testing.T) {
 			nginx.store = fakeIngressStore{
 				ingresses: []*ingress.Ingress{},
 				configuration: ngx_config.Configuration{
-					DisablePathTypeValidation:  true,
+					EnablePathTypeValidation:   false,
```

Comment:
https://github.com/kubernetes/ingress-nginx/blob/4b86945d1015d274a4d065314618b20ba8144990/internal/ingress/controller/controller_test.go#L290

This case should return an error, what do you think 🤔